easybuild/easyconfigs/c/Catch2/Catch2-3.3.2-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'CMakeMake'
```

Comment:
Does this need cmakemake easyblock? If truly so, then it would also need a CMake builddep. 

If it's under GCCcore it also needs binutils as a builddep (to make the test CI suite happy)

Reply:
Catch2 does have a CMakeLists.txt, which allows for installation.
Added the deps